tools/dotcms-cli/cli/src/test/java/com/dotcms/cli/command/PushCommandIT.java

```diff
@@ -115,6 +115,39 @@ void testSimplePush() throws IOException {
         }
     }
 
+    /**
+     * This test checks for a simple push situation using a relative path where everything should work as expected.
```

Comment:
can you please add a Given Scenario:  and then the scenario in which the test is supposed to happen
and then the Expected Result:  and the the result of the test
I know it is a bit painful but we're trying to be consistent across all our test documentation wise

Reply:
Yes, for sure. No problem